winui/AvatarView/getting-started.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Getting Started with WinUIAvatar View control | Syncfusion
+description: Learn all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more here.
```

Comment:
Description: Learn about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more.